cmd/root.go

```diff
@@ -70,7 +72,7 @@ func initConfig() {
 		// Find home directory.
 		home, err := homedir.Dir()
 		if err != nil {
-			fmt.Println(err)
+			log.Fatal(err)
 			os.Exit(1)
```

Comment:
get rid of `os.Exit` log.Fatal will exit for you

Reply:
:+1: